pkg/chart/chart.go

```diff
@@ -424,8 +424,13 @@ func (t *Testing) ComputeChangedChartDirectories() ([]string, error) {
 		}
 		dir := filepath.Dir(file)
 		// Only add if not already in list and double-check if it is a chart directory
-		if !util.StringSliceContains(changedChartDirs, dir) && t.chartUtils.IsChartDir(dir) {
-			changedChartDirs = append(changedChartDirs, dir)
+		if !util.StringSliceContains(changedChartDirs, dir) {
```

Comment:
Need to change order here and check if `chartDir` is added already. Otherwise we will get multiple entries if there are multiple files changed.

Reply:
This should work:
```go
		chartDir, err := t.chartUtils.LookupChartDir(cfg.ChartDirs, dir)
		if err == nil {
			if !util.StringSliceContains(changedChartDirs, chartDir) {
				changedChartDirs = append(changedChartDirs, chartDir)
			}
		} else {
			fmt.Printf("Directory '%s' is not chart directory. Skipping...", chartDir)
		}
```

Reply:
Good catch. Thanks.

Reply:
Fixed as suggested. I figured more elaborate test would no be easy. I'd leave them as is for now.